packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -23,6 +23,14 @@ const ModuleScopePlugin = require('react-dev-utils/ModuleScopePlugin');
 const paths = require('./paths');
 const getClientEnvironment = require('./env');
 
+// Get supported browsers list
```

Comment:
I think this can be moved to a separate module to avoid DRY, maybe config/getSupportedBrowsers?